modules/dbcan.nf

```diff
@@ -36,19 +36,27 @@ process DBCAN {
             echo "\$line"
         fi
     done < ${gff} > ${cluster_name}_noseq.gff
-
+    
+    args=""
+    if [ ${domain} != "Eukaryota" ]; then
```

Comment:
I think you can actually set this in the config similar to https://github.com/EBI-Metagenomics/genomes-catalogue-pipeline/blob/5b69dfcc30208be7cc666fbdacdc0df202faec66/nextflow.config#L59 if you think it's cleaner. But this is also perfectly fine

Reply:
Yeah, the drep example is nice. I think it would be slightly messier here because for both proks and euks dbcan is run from the same subworkflow. I will leave it for now but may come back to it later.